app/config/play_ui/AssetsConfig.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config.play_ui
+
+import javax.inject.Inject
+import play.api.Configuration
+
+class AssetsConfig @Inject()(configuration: Configuration) {
+  private val assetsUrl = configuration.get[String]("assets.url")
+  private val assetsVersion = configuration.get[String]("assets.version")
+  val assetsPrefix: String = assetsUrl + assetsVersion
```

Comment:
Hi Don. Bit confused, seems like you've added in assets-frontend stuff that should be being removed as a dependency no?

Reply:
Unfortunately true, there were no direct equivalent for the majority of the components from play-ui in the play-frontend-hmrc library and this was how CEDS-3314 had been addressed.